examples/basic/example.tf

```diff
@@ -27,3 +27,52 @@ module "prober" {
   importpath  = "github.com/chainguard-dev/terraform-google-prober/examples/basic"
   working_dir = path.module
 }
+
+// Create an alert policy based on the uptime check above.
+resource "google_monitoring_alert_policy" "prober_uptime" {
+  project = var.project_id
+  # In the absence of data, incident will auto-close in 7 days
+  alert_strategy {
+    auto_close = "604800s"
+  }
+  combiner = "OR"
+
+  conditions {
+    condition_threshold {
+      aggregations {
+        alignment_period     = "300s"
+        cross_series_reducer = "REDUCE_COUNT_FALSE"
+        group_by_fields      = ["resource.*"]
+        per_series_aligner   = "ALIGN_NEXT_OLDER"
+      }
+
+      comparison = "COMPARISON_GT"
+      duration   = "60s"
+      filter     = format("metric.type=\"monitoring.googleapis.com/uptime_check/check_passed\" resource.type=\"uptime_url\" metric.label.\"check_id\"=\"%s\"", module.prober.uptime_check)
+
+      threshold_value = "1"
+
+      trigger {
+        count   = "1"
+        percent = "0"
+      }
+    }
+
+    display_name = "${module.prober.uptime_check_name} probe failure"
+  }
+
+  display_name = "${module.prober.uptime_check_name} prober failed alert"
+  enabled      = "true"
+
+  documentation {
+    content = <<-EOT
+    Please consult the playbook entry [here](https://chainguard-dev.com) for how to respond.
```

Comment:
Maybe change this to just be a note that folks can link to a playbook entry instead?